frontend/kendra-button-service/kendra.js

```diff
@@ -1,3 +1,7 @@
+import { Logger } from 'aws-amplify';
```

Comment:
This file is vanilla script so please don't import this Logger and remove other related `Logger` code.

Reply:
Thank you for letting me know. I'm on it!